app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
```

Comment:
You are expected to modify the devise form in order to make it centrally aligned in order to match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png)

Also, when I create a new account, it requests verification which is expected but according to the given instructions, I was to receive a mail that I can use to confirm my details before logging in I have been waiting for that mail now but still, I did not receive it, pls ensure you have the correct set up for that (you can utilize gems like `letter opener` to do so) to enable users access to your app.